core/domain/services/pue/Config.php

```diff
@@ -58,7 +58,7 @@ public function checkPeriod()
 
     public function optionKey()
     {
-        return 'site_license_key';
+        return 'ee_site_license_key';
```

Comment:
we don't need to migrate over the old option, right? The old option (which may have been hijacked by another plugin) just isn't used anymore, right? What will the code do when it can't find the old `site_license_key` WP option?

Reply:
The name for this is actually a bit misleading.  When PUE was first implemented, there was actually a specific option using this key in the option table and thus saving the form with the input used the option key as the input `name` attribute.  Since then, we still have a form input using this value as its `name` attribute, but the actual value is saved in the database as a part of `Network_Config`.

Reply:
ahhh, gotcha. So changing this only affects the form, not how the data is stored. Sounds good.